std/net/curl.d

```diff
@@ -627,6 +627,60 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
+ */
+private string urlEncode()(in string[string] values)
+{
+    bool first = true;
+    Appender!string enc;
+    enc.reserve(values.length * 512);
+    foreach (k, v; values)
+    {
+        if (!first)
+            enc ~= '&';
+        import std.uri : encodeComponent;
+
+        enc ~= encodeComponent(k);
+        enc ~= "=";
+        enc ~= encodeComponent(v);
+        first = false;
+    }
+    return enc.data;
+}
+
+/** HTTP post content.
+ *
+ * Params:
+ * url = resource to post to
+ * postData = data to send as the body of the request. An associative array of $(D string) is accepted and will be
+ *            encoded using www-form-urlencoding
+ * conn = HTTP connection to use
+ *
+ * Returns:
+ * A $(D char[]) containing the content of the resource pointed to by the URL.
+ *
+ * See_Also: $(LREF HTTP.Method)
+ */
```

Comment:
please use `/// ditto` - ddoc will then group it and you don't have to copy the ddoc header.
Moreover it's nicer for the user.
Btw ddoc is a bit "smart", it will only display the actual parameters for the the overload then. So just name them differently - e.g `postValues` or `postDict` and it should work.